<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GTool</class>
 <widget class="QMainWindow" name="GTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QTabWidget" name="FunctionWgedit">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="FirstForm" name="First">
       <attribute name="title">
        <string>First</string>
       </attribute>
      </widget>
      <widget class="OtherForm" name="Other">
       <attribute name="title">
        <string>Other</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="LogForm" name="Log" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLog">
    <property name="title">
     <string>Log</string>
    </property>
    <addaction name="actionSingle"/>
    <addaction name="separator"/>
    <addaction name="actionInfo"/>
    <addaction name="actionDebug"/>
    <addaction name="actionWarn"/>
    <addaction name="actionError"/>
    <addaction name="separator"/>
    <addaction name="actionOtherMessage"/>
   </widget>
   <addaction name="menuLog"/>
  </widget>
  <action name="actionInfo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionWarn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Warn</string>
   </property>
  </action>
  <action name="actionError">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Error</string>
   </property>
  </action>
  <action name="actionOtherMessage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>OtherMessage</string>
   </property>
  </action>
  <action name="actionSingle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Single</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LogForm</class>
   <extends>QWidget</extends>
   <header>LogForm.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OtherForm</class>
   <extends>QWidget</extends>
   <header>OtherForm.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FirstForm</class>
   <extends>QWidget</extends>
   <header>FirstForm.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
